Build Error on windows

Cannot build java nats on windows:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (prepare) on project jnats: An Ant BuildException has occured: **Can't get https://github.com/nats-io/g
natsd/releases/download/v0.9.4/gnatsd-v0.9.4-windows-${nats.arch}.zip to D:\EclipseWorkspace\HouseControlCommunication\jnats\java-nats-0.5.3\target\gnatsd-v0.9.4-windows-${nats.arch}.zi**
p
[ERROR] around Ant part ...<get skipexisting="true" src="https://github.com/nats-io/gnatsd/releases/download/v0.9.4/gnatsd-v0.9.4-windows-${nats.arch}.zip" dest="D:\EclipseWorkspace\Hou
seControlCommunication\jnats\java-nats-0.5.3\target" unless:true="${server.installed}"/>... @ 9:249 in D:\EclipseWorkspace\HouseControlCommunication\jnats\java-nats-0.5.3\target\antrun\
build-copy-server-exec.xml
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

--- a/pom.xml
+++ b/pom.xml
@@ -100,15 +100,6 @@
         <!-- Only unit tests are run by default. -->
         <skip.integration.tests>true</skip.integration.tests>
         <skip.unit.tests>false</skip.unit.tests>
-
-        <!-- Used for downloading and installing a server binary for tests -->
-        <server.version>0.9.6</server.version>
-        <server.exec.name>gnatsd</server.exec.name>
-        <server.exec.pkg>${server.exec.name}-v${server.version}-${nats.os}-${nats.arch}
-        </server.exec.pkg>
-        <server.exec.pkg.url>
-            https://github.com/nats-io/${server.exec.name}/releases/download/v${server.version}/${server.exec.pkg}.zip
-        </server.exec.pkg.url>
     </properties>
 
     <build>
@@ -177,21 +168,6 @@
                 </configuration>
                 <executions>
                     <execution>
-                        <id>prepare</id>
-                        <phase>pre-integration-test</phase>
-                        <configuration>
-                            <target xmlns:unless="ant:unless" xmlns:if="ant:if" name="copy-server-exec" unless:true="${server.installed}">
-                                <condition property="server.installed" else="false">
-                                    <available file="${server.exec.name}" filepath="${project.build.directory}" type="file" />
-                                </condition>
-                                <echo if:true="${server.installed}" message="${server.exec.name} is already installed. Skipping download..." />
-                                <echo unless:true="${server.installed}" message="Installing ${server.exec.pkg} as ${project.build.directory}/${server.exec.name}..." />
-                                <get unless:true="${server.installed}" src="${server.exec.pkg.url}" dest="${project.build.directory}" skipexisting="true" />
-                                <unzip unless:true="${server.installed}" src="${project.build.directory}/${server.exec.pkg}.zip" dest="${project.build.directory}" overwrite="false" />
-                                <move unless:true="${server.installed}" file="${project.build.directory}/${server.exec.pkg}/${server.exec.name}" tofile="${project.build.directory}/${server.exec.name}" overwrite="false" />
-                                <chmod perm="+x" file="${project.build.directory}/${server.exec.name}" />
-                            </target>
-                        </configuration>
                         <goals>
                             <goal>run</goal>
                         </goals>

--- a/src/test/java/io/nats/client/NatsServer.java
+++ b/src/test/java/io/nats/client/NatsServer.java
@@ -88,8 +88,9 @@ class NatsServer implements Runnable, AutoCloseable {
     }
 
     private ProcessStartInfo createProcessStartInfo() {
-        String path = Paths.get("target", "/", GNATSD).toAbsolutePath().toString();
-        psInfo = new ProcessStartInfo(path);
+
+        // The NATS server must be on the path.
+        psInfo = new ProcessStartInfo(GNATSD);
 
         if (debug) {
             psInfo.addArgument("-DV");
@@ -106,15 +107,26 @@ class NatsServer implements Runnable, AutoCloseable {
                 System.err.println("Inheriting IO, psInfo =" + psInfo);
                 pb.inheritIO();
             } else {
-                pb.redirectError(new File("/dev/null"));
-                pb.redirectOutput(new File("/dev/null"));
+                // All NATS server output goes to stderr
+                String errFile = "/dev/null";
+
+                String osName = System.getProperty("os.name");
+                if (osName != null && osName.contains("Windows")) {
+                    // Windows uses the "nul" file.
+                    errFile = "nul";
+                }
+
+                pb.redirectError(new File(errFile));
             }
             p = pb.start();
             if (debug) {
                 System.out.println("Started [" + psInfo + "]");
             }
         } catch (IOException e) {
-            e.printStackTrace();
+            System.out.println("Unable to start [" + psInfo + "]. The NATS server " +
+                    "must be installed and found in the path.\n" +
+                    "See https://github.com/nats-io/gnatsd");
+            System.out.println(e.getMessage());
         }
     }
 

